<launch>
  <node pkg="moveit_ros_move_group" type="move_group" name="any_name" output="screen">
    <remap from="joint_states" to="robot/joint_states"/>
  </node>
  <node name="pose" pkg="armbot_move" type="pose" respawn="false" output="screen">
    <rosparam command="load" file="$(find armbot_moveit_config)/config/kinematics.yaml"/>
    <param name="planning_plugin" value="ompl_interface/OMPLPlanner"/>
    <rosparam command="load" file="$(find armbot_moveit_config)/config/ompl_planning.yaml"/>
  </node>
</launch>